### PR TITLE
Add rakefile

### PR DESCRIPTION
**Pull request by jeff1108**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -23,12 +23,23 @@ task :setup do
...
+      password VARCHAR(60)
+      );")
     connection.exec("CREATE TABLE listings (
       id SERIAL PRIMARY KEY,
       name VARCHAR(60),
       description VARCHAR(350),
-      price INT4
+      price INT4,
+      available BOOL,
+      requested BOOL,
```
Comment (heuperman) [inline Rakefile]: change to:
available BOOL DEFAULT true,
requested BOOL DEFAULT false,

File: Rakefile [hunk 2]
```diff
@@ -50,18 +61,25 @@ task :populate do
...
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('4 Privet Drive', 'A roomy cosy space under staircase', 25, FALSE, FALSE, 2);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Azkaban cell', 'No frills room available all year round!', 10, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Gryffindor House', 'A lovely warm space, would suit family of 300', 100, FALSE, FALSE, 3);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Slytherin House', 'A welcoming space, would suit ex-murders', 50, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Hufflepuff House', 'A spacious place, great for all types of people', 75, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Ravenclaw House', 'A warm and cozy space, would suit family of 100', 75, FALSE, FALSE, 4);")
```
Comment (heuperman) [inline Rakefile]: Don't need the FALSE, FALSE  on each of these if we set default values (also remove available, requested from INSERT INTO).

Comment (heuperman) [reply Rakefile]: We want spaces to be available by default?

**Comment by jeff1108** [comment]
Done (Added the default value of available and requested)

**Pull request merged**